Sniffs.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Sniffs;
+
+class Sniffs
+{
+    const DEFAULT_POINTS = 30000;
```

Comment:
The base remediation points is 50,000, which we consider to be something that takes 5 minutes to resolve. IMO, I would bump everything up by 40k, making the ones you marked as 10k 50k, and the default 70k